controller/version.go

```diff
@@ -0,0 +1,13 @@
+package controller
+
+import (
+	"context"
+
+	"github.com/google/go-github/github"
+)
+
+func (c *Controller) GetLatestVersion() (string, error) {
+	client := github.NewClient(nil)
```

Comment:
With the idiom we're using, client should be init'd inside the controller's New function

Whether that's the right idiom, for this, I'm growing to think not